docs/docsite/rst/dev_guide/developing_locally.rst

```diff
@@ -5,9 +5,11 @@
 Adding modules and plugins locally
 **********************************
 
-The easiest, quickest, and the most popular way to extend Ansible is to use a local module or a plugin. You can create them or copy existing ones for local use. You can store a local module or plugin on your Ansible control node and share it with your team or organization. You can also share a local plugin or module by including it in a collection or embedding it in a role, then publishing the collection or role on Ansible Galaxy. If you are using roles on Ansible Galaxy, then you are already using local modules and plugins without realizing it.
+The easiest, quickest, and the most popular way to extend Ansible is to use a local module or a plugin. You can create them or copy existing ones for local use. You can store a local module or plugin on your Ansible control node and share it with your team or organization. You can also share a local plugin or module by including it in a collection, then publishing the collection on Ansible Galaxy. If you are using collections from Ansible Galaxy, then you are already using local modules and plugins without realizing it.
 
-If you are using an existing module or plugin but Ansible can't find it, this page is all you need. However, if you want to create a plugin or a module, go to :ref:`developing_plugins` and :ref:`developing_modules_general` topics and then return to this page to know how to add it locally.
+If you are using a legacy, standalone module or plugin but Ansible can't find it, this page is all you need.
```

Comment:
The introductory paragraph says 

> The easiest, quickest, and the most popular way to extend Ansible is to use a local module or a plugin [...]

but here we're saying local plugins are legacy.   Perhaps we want to use less enthusiastic language in the introductory paragraph?  Or maybe we don't want to use legacy here (as collections serve a different niche than local plugins)?

Reply:
addressed in c3790e3887d1eb3d7b3437f5ec4794233f1ff03d